rpmconf/Install/Util/modules/install_rabbitmq.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
```

Comment:
a little bit confused if https://github.com/Zimbra/zm-build/pull/228/files#diff-959e3a8de3363d97be718a30338f1e76e23fce4f4030da828aae140cc039a19cR2487 line is already installing the package then why do we need this file?

Reply:
not required, committed changes for the same